google/cloud/aiplatform/jobs.py

```diff
@@ -1910,3 +1919,448 @@ def run(
+                ):
+                    raise RuntimeError(
+                        "Invalid config for model. `explanation_config` should only be enabled if the model has `explanation_spec populated"
+                    )
+                all_configs.append(
+                    gca_model_deployment_monitoring_job.ModelDeploymentMonitoringObjectiveConfig(
+                        deployed_model_id=key,
+                        objective_config=objective_configs[key].as_proto(),
+                    )
+                )
+
+        return all_configs
+
+    @classmethod
+    def create(
+        cls,
+        endpoint: Union[str, "aiplatform.Endpoint"],
+        objective_configs: Union[
+            model_monitoring.EndpointObjectiveConfig,
+            Dict[str, model_monitoring.EndpointObjectiveConfig],
+        ],
+        logging_sampling_strategy: model_monitoring.RandomSampleConfig,
+        schedule_config: model_monitoring.ScheduleConfig,
+        timeout: float = None,
+        display_name: Optional[str] = None,
+        deployed_model_ids: Optional[List[str]] = None,
+        alert_config: Optional[model_monitoring.EmailAlertConfig] = None,
+        predict_instance_schema_uri: Optional[str] = None,
+        sample_predict_instance: Optional[str] = None,
+        analysis_instance_schema_uri: Optional[str] = None,
+        bigquery_tables_log_ttl: Optional[int] = None,
+        stats_anomalies_base_directory: Optional[str] = None,
+        enable_monitoring_pipeline_logs: Optional[bool] = None,
+        labels: Optional[Dict[str, str]] = None,
+        encryption_spec_key_name: Optional[str] = None,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ) -> "ModelDeploymentMonitoringJob":
+        """Creates and launches a model monitoring job
+
+        Args:
+            endpoint (Union[str, "aiplatform.Endpoint"]):
+                Endpoint resource name. Format:
+                ``projects/{project}/locations/{location}/endpoints/{endpoint}``
+
+            objective_configs (Union[
+                model_monitoring.objective.EndpointObjectiveConfig,
+                Dict[str, model_monitoring.objective.EndpointObjectiveConfig]):
+                A single config if it applies to all models, or a dictionary of
+                model_id: model_monitoring.objective.EndpointObjectiveConfig if
+                different model IDs have different configs
+
+            logging_sampling_strategy (model_monitoring.sampling.RandomSampleConfig):
+                Sample Strategy for logging.
+
+            schedule_config (model_monitoring.schedule.ScheduleConfig):
+                Configures model monitoring job scheduling interval in hours.
+                This defines how often the monitoring jobs are triggered.
+
+            timeout (float): timeout for the model monitoring job creation request
```

Comment:
Since this is a required argument, it needs to come before the optional arguments. In that case I will rename the argument, mark it optional, and move it to the bottom.